lib/event_bridge.rb

```diff
@@ -0,0 +1,31 @@
+class EventBridge
+
+  def put_event(event_name, payload)
+    client = Aws::EventBridge::Client.new(
+      credentials: Aws::Credentials.new(
+        ENV['???'],
+        ENV['???']
+      )
+      region: ENV['???']
+    )
+
+    resp = client.put_events(entries: [
```

Comment:
source - I really wanted this to be the sending service - meta.shift72.com
resources - null
detail_type - event_name
detail - payload.to_json
event_bus_name - this needs to be an ARN to an event bus of the publishing service, has a relationship with the source
track_header: null

For example
the event_name - `meta_item_added`   -  
```
::EventBGBus.publish('meta_item_created',
    {
      client_id: client_id,
      slug: slug,
      title: title,
      tagline: meta_data['tagline'],
      genres: extract_genres,
      event_date: Time.now.utc,
      external_id: external_id,
      external_id2: external_id2,
      status: status_id == 2 ? "published" : "draft"
    })
```
Needs to publish to
```
source : "meta.shift72.com",   // we will in the future be publishing some of these messages externally
event_bus_name: `arn:aws:events:ap-southeast-2:494740357900:event-bus/meta` (does not exist yet)
detail_type: "meta_item_created",
detail: msg.to_json
```

Reply:
For the `source` and `event_bus_name` options... do you reckon they should be passed from the calling service via `EventBGBus.publish`?

Reply:
@sarge ?

Reply:
@sarge Hey dude, just this pull requests :shipit: !! Just kidding, hope u are well, I am using AWS Event Bridge on my work, very good service.

Anyways, feel free to remove me from this repo's access, I just checked and I cant see any private repos.